pkg/controller/seed-controller-manager/seedresourcesuptodatecondition/seedresourcesuptodatecondition_controller.go

```diff
@@ -182,5 +181,5 @@ func (r *reconciler) seedResourcesUpToDate(ctx context.Context, cluster *kuberma
 		}
 	}
 
-	return true, nil
+	return len(deployments.Items) > 0 || len(statefulSets.Items) > 0, nil
```

Comment:
Add a comment here, and clarify that this is to avoid adding the resources up-to-date condition in the cluster before deployments end ss are applied.